src/components/Comments/ReplyCommentItem.js

```diff
@@ -314,11 +377,18 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition to display the total vote count has been removed. This change seems unnecessary and could potentially lead to displaying a zero vote count, which might not be desirable from a UX perspective.
```diff
- {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+ {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->